index.html

```diff
@@ -16,12 +16,22 @@
         <nav class="hamburger">
                 <a class="logo" href="#">Welcome</a>
                 <ul class="link-desktop">
-                    <li class="links-style"><a href="#">Portoflio</a></li>
-                    <li class="links-style"><a href="#">About</a></li>
-                    <li class="links-style"><a href="#">Contact</a></li>
+                    <li class="links-style"><a href="#works">Portoflio</a></li>
+                    <li class="links-style"><a href="#about">About</a></li>
+                    <li class="links-style"><a href="#contact">Contact</a></li>
                 </ul>
```

Comment:
Hey Team, please, see these lines of [codes in your HTML file](https://github.com/Ferizad/Portfolio/blob/f4d6d86a92242474ce3008c6c664610b29e0a11e/index.html#L33-L51), there's a duplicate in your mobile menu links, please, kindly remove the duplicates